File: app-apple/Sources/AppLibrary/Theme/Views/ThemeLogo.swift
// SPDX-FileCopyrightText: 2026 Davide De Rosa
//
// SPDX-License-Identifier: GPL-3.0

import SwiftUI

public struct ThemeLogo: View {

    @Environment(Theme.self)
    private var theme

    public init() {
    }

    public var body: some View {
        Image(theme.logoImage)
    }
}

File: app-android/app/src/main/kotlin/com/algoritmico/passepartout/ui/theme/ThemeLogo.kt
// SPDX-FileCopyrightText: 2026 Davide De Rosa
//
// SPDX-License-Identifier: GPL-3.0

package com.algoritmico.passepartout.ui.theme

import androidx.compose.foundation.Image
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.res.painterResource
import com.algoritmico.passepartout.R

@Composable
fun ThemeLogo(
    modifier: Modifier = Modifier
) {
    Image(
        painter = painterResource(R.drawable.app_logo),
        contentDescription = null,
        modifier = modifier
    )
}
